src/site/content/en/blog/cls-web-tooling/index.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Evolving Cumulative Layout Shift in Web Tooling
```

Comment:
```suggestion
title: Evolving Cumulative Layout Shift in web tooling
```

Reply:
@rachelandrew Please check that titles and headings use sentence case: https://web.dev/handbook/grammar/#titles-and-headings